components/core/src/clp_s/Defs.hpp

```diff
@@ -1,46 +1,32 @@
-// Code from CLP
-
 #ifndef CLP_S_DEFS_HPP
 #define CLP_S_DEFS_HPP
 
 // C++ libraries
-#include <atomic>
-#include <cinttypes>
 #include <cstdint>
 #include <limits>
 
 namespace clp_s {
 // Types
 using epochtime_t = int64_t;
-static epochtime_t const cEpochTimeMin = INT64_MIN;
-static epochtime_t const cEpochTimeMax = INT64_MAX;
-static double const cDoubleEpochTimeMin = std::numeric_limits<double>::lowest();
-static double const cDoubleEpochTimeMax = std::numeric_limits<double>::max();
-#define SECONDS_TO_EPOCHTIME(x) x * 1000
-#define MICROSECONDS_TO_EPOCHTIME(x) 0
-#define EPOCHTIME_T_PRINTF_FMT PRId64
+constexpr epochtime_t cEpochTimeMin{INT64_MIN};
+constexpr epochtime_t cEpochTimeMax{INT64_MAX};
```

Comment:
```suggestion
constexpr epochtime_t cEpochTimeMin{std::numeric_limits<epochtime_t>::min()};
constexpr epochtime_t cEpochTimeMax{std::numeric_limits<epochtime_t>::max()};
```

I wouldn't normally comment on something like this since it's not a clang-tidy violation; but at the same time, it's a small fix and not one we're likely to catch as part of another clean-up effort, so I think it's worth doing now.